core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2666,4 +2666,13 @@ package object config {
       .version("4.0.0")
       .booleanConf
       .createWithDefault(false)
+
+  private[spark] val SPARK_SHUTDOWN_TIMEOUT_MS =
+    ConfigBuilder("spark.shutdown.timeout")
```

Comment:
Please add the following to make this internal configuration, @robreeves .
```
.internal()
```

Reply:
Updated